README.md

```diff
@@ -113,12 +113,12 @@ Have a bug or a feature request? Please first read the [issues guidelines](https
 
 ## Documentation
 
-Official plotly.js documentation is hosted on [plot.ly/javascript](https://plot.ly/javascript).
+Official plotly.js documentation is hosted on [plotly.com/javascript](https://plotly.com/javascript).
 
-These pages are generated by the Plotly [documentation repo](https://github.com/plotly/documentation/tree/gh-pages) built with [Jekyll](http://jekyllrb.com) and publicly hosted on GitHub Pages.
+These pages are generated by the Plotly [documentation repo](https://github.com/plotly/documentation/tree/gh-pages) built with [Jekyll](https://jekyllrb.com/) and publicly hosted on GitHub Pages.
 For more info about contributing to Plotly documentation, please read through [contributing guidelines](https://github.com/plotly/documentation/blob/source/Contributing.md).
 
-You can also suggest new documentation examples by submitting a [Codepen](http://codepen.io/tag/plotly/) on community.plot.ly with tag [`plotly-js`](http://community.plot.ly/c/plotly-js).
+You can also suggest new documentation examples by submitting a [Codepen](https://codepen.io/tag/plotly/) on community.plot.com with tag [`plotly-js`](https://community.plotly.com/c/api/plotly-js).
```

Comment:
Let's remove this line plz

Reply:
Done in https://github.com/plotly/plotly.js/pull/4694/commits/1fe9c53d66973721d0a512c4570b9c7d612b1f90.